dist/src/core/mathAtom.js

```diff
@@ -751,32 +751,34 @@ class MathAtom {
         // First, we do the same steps as in overline to build the inner group
         // and line
         const inner = decompose(context.cramp(), this.body);
-        const ruleWidth = FONTMETRICS.defaultRuleThickness /
-            mathstyle.sizeMultiplier;
+        const ruleWidth = FONTMETRICS.defaultRuleThickness * mathstyle.sizeMultiplier;
         let phi = ruleWidth;
         if (mathstyle.id < Mathstyle.TEXT.id) {
             phi = mathstyle.metrics.xHeight;
         }
         // Calculate the clearance between the body and line
         let lineClearance = ruleWidth + phi / 4;
-        const innerTotalHeight = Math.max(2 * phi, (Span.height(inner) + Span.depth(inner)) * mathstyle.sizeMultiplier);
-        const minDelimiterHeight = innerTotalHeight + (lineClearance + ruleWidth);
+        const innerSize = Span.height(inner) + Span.depth(inner);
+        const innerSizeMin = Math.max(2 * phi, innerSize * mathstyle.sizeMultiplier);
+        const minDelimiterHeight = innerSizeMin + (lineClearance + ruleWidth);
 
         // Create a \surd delimiter of the required minimum size
         const delim = makeSpan(Delimiters.makeCustomSizedDelim('', '\\surd', minDelimiterHeight, false, context), 'sqrt-sign');
         delim.applyStyle(this.getStyle());
 
-        const delimDepth = (delim.height + delim.depth) - ruleWidth;
+        // The size of the delimiter that extends below the sqrt-line
+        const visibleDelimSize = ((delim.height + delim.depth) * mathstyle.sizeMultiplier) - ruleWidth;
```

Comment:
some of the multiplier logic in here was incorrect or missing. after lots of fiddling, fixing the ruleWidth and visibleDelimSize scaling seemed the best option, e.g. for this scaled down sqrt

![image](https://github.com/purdue-tlt/mathlive/assets/689209/b46e9c1d-e209-4b71-bd39-8cc7aa297782)